lib/view/edit_item_dialog.dart

```diff
@@ -42,15 +52,17 @@ class _EditItemDialogState extends State<EditItemDialog> {
             },
           ),
           const SizedBox(height: 16),
-          _buildItemTypeSwitcher()
+          _buildItemTypeSwitcher(),
+          const SizedBox(height: 8),
+          _buildRepeatWidget(),
+          // _buildRepeatOptions()
```

Comment:
Should this still be there?

Reply:
No, that definitely should not be there. That was my mistake.

Reply:
No worries! It happens all the time